src/components/organisms/shared/OakMediaClipStackListItem/OakMediaClipStackListItem.tsx

```diff
@@ -110,6 +113,9 @@ export const OakMediaClipStackListItem = (
           $height={["all-spacing-11", "all-spacing-11", "all-spacing-15"]}
           $position={"relative"}
           $mb={["space-between-none", "space-between-none", "space-between-xs"]}
+          $showOakPlaceholder={!imageUrl ? true : false}
+          src={imageUrl ?? oakPlaceholder}
+          alt={!imageUrl ? "oak" : ""}
```

Comment:
might be weird when screen reader reads out 'oak'? 🤔 I guess leave it an empty string in any case as these images don't carry any info or  make it "placeholder image for video" in both cases